Week-2/Assignment2.1/webpage.css

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
You should have used grid for this assignment

Reply:
Also need to use media queries for responsiveness

Reply:
Actually we did not have recording sessions on grid so i was not familiar with grids so did with flex..